plugins/BEdita/Core/config/Migrations/20171104185428_ObjectTypesKey.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Cake\ORM\Table;
+use Migrations\AbstractMigration;
+
+/**
+ * Fix `object_type_id` foreign key index.
+ */
+class ObjectTypesKey extends AbstractMigration
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function up()
+    {
+        $this->table('objects')
+            ->dropForeignKey(
+                'object_type_id'
+            )
+            ->update();
+
+        $this->table('objects')
+            ->addForeignKey(
+                'object_type_id',
+                'object_types',
+                'id',
+                [
+                    'constraint' => 'objects_objtype_fk',
+                    'update' => 'NO_ACTION',
+                    'delete' => 'RESTRICT'
+                ]
+            )
+            ->update();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function down()
+    {
```

Comment:
This method shouldn't be empty… it should revert changes made in `up()`. In this case it is not _strictly_ necessary since the migration is idempotent, but I'd prefer to have it anyways — and in any case it would probably be better to at least leave a comment to explain why it is being left empty.

Reply:
:+1:  for an explanatory comment